### ISSUE TITLE
how to fine-tune in V100 x 8 

### ISSUE THREAD

**Issue by Morxrc** [tatsu-lab/stanford_alpaca#52]
need help!

**Comment by KurtFeynmanGodel** [comment]
Reduce batch sizes and gradient accumulation. See if that helps.

**Comment by Morxrc** [comment]
> The be more specific, you need to reduce the batch size and INCREASE the gradient accumulation correspondingly.

batchsize 1 OOM.
